src/de/grobox/liberario/TransportNetwork.java

```diff
@@ -57,6 +57,12 @@ public TransportNetwork setDescription(@Nullable String description) {
 		return this;
 	}
 
+	public TransportNetwork setDescription(@Nullable String description, @Nullable String networks) {
```

Comment:
What happens when the current language has no translation? Does the context then contain simply the original string?
I think both variables do not need to be Nullable. When we pass on null it just looks awkward, but nothing "bad" happens.